### DISCUSSION TITLE
Top growing pages

### DISCUSSION THREAD

**Discussion by Cdingram** [plausible/analytics#2099] [Anything Else]
It would be super useful to be able to see which pages have had the biggest increase in visitors for the timeframe selected. Top pages are useful but in order to identify what's working in my growth strategy and what pages should be nutured, it would be very helpful to see what pages have had the largest increase (and for that matter decrease) in popularity.

Google search console has something like this in the monthly insights where they show top growing pages (they use a "+123 visitors" but perhaps a % increase would be better? Not Sure). 

This is sort of similar to several discussions on adding a previous timeframe line to the chart, but I think it's useful on its own to see a quick view of the pages that have improved the most. 

Not sure the best algorithm to do this off the top of my head, but dealing with new pages for example would have to be considered (a new page with one view shouldn't be percieved as 100% increase, perhaps this is why google used the actual +xx visitors count rather than %).

Thanks for the great work as always!

**Comment by RobertJoonas** [comment]
Thanks for the feature request @Cdingram!

Link to main discussion on the same topic: https://github.com/plausible/analytics/discussions/2097.

I think it makes sense to keep that other thread as the main one for this discussion, as it's a bit broader and mentions other properties than pages too. Thanks!
